packages/db/prisma/schema.prisma

```diff
@@ -185,15 +183,30 @@ model Workout {
 // * GROUP: Group message
 // ******************************************************************************************
 model Exercise {
-  id        Int       @id @unique @default(autoincrement())
-  name      String    @db.Char(100)
-  note      String?
+  id           Int           @id @unique @default(autoincrement())
+  name         String        @db.Char(100)
+  note         String?
+  bodyPart     BodyPart
+  category     Category
+  exerciseLogs ExerciseLog[]
+
+  user   User? @relation(fields: [userId], references: [id])
```

Comment:
now each exercise link to each user, what make it differences from the exercise log?

Reply:
I think you trynna convey the logic to me, but now I'm confused.

~~Where would be our data bank for the exercise (the "template" that they choose from)~~

Reply:
now if the `workoutTemplate` links exercise through `exercisesIds`.

Now there will be an exercise where it link to the non-existing user. and we treat those as exercise bank?